modules/gluonw-base/src/main/scala/gluonw/common/GluonWAlgorithm.scala

```diff
@@ -103,15 +105,33 @@ case class GluonWConstants(precision: Long = GluonWBoxConstants.PRECISION)
   }
 
   def neutronsToNanoErg(
+    neutronsInCirculation: Long,
     neutronsAmount: Long,
+    fissionedErg: Long,
     goldPriceGramsNanoErg: Long
-  ): Long =
-    (BigInt(neutronsAmount) * BigInt(goldPriceGramsNanoErg) / GluonWBoxConstants.PRECISION).toLong
+  ): Long = {
+    val fusRation: BigInt =
+      fusionRatio(
+        neutronsInCirculation,
+        goldPriceGramsNanoErg,
+        fissionedErg
+      )
+
+    // Equations for determining the neutron price, Pn, and neutron volume, Vn.
```

Comment:
These commnets are very nice. I suggest adding the to the contract as well.

Reply:
Ok, I added them.